tests/integration/run_tests.py

```diff
@@ -52,7 +52,8 @@
     ("analysis", "ERZ266973.json", "ERZ266973.json"),
 ]
 base_url = os.getenv("BASE_URL", "http://localhost:5430")
-mock_auth_url = os.getenv("OIDC_URL", "http://localhost:8000")
+# mock_auth_url = os.getenv("OIDC_URL", "http://localhost:8000")
```

Comment:
that means this OIDC_URL is set somewhere ? - why do we need to comment it out here?
I see in the github actions is set to localhost:8000 and i assume locally will work with default value.

Reply:
There was a problem, where the integration test would not run, because it was started from outside, and `OIDC_URL` was set to `mockauth` which was only resolvable from inside the docker-network. I differentiated these now to two different URLs, which can also be used in the OIDC config
https://github.com/CSCfi/metadata-submitter/pull/315/commits/8d81026c4e5704653aff390da3abba8050781225#diff-9049ef8d86388471b49a45e70f5427febf75a8d5fb0873132422e2f132b89d34R47-R48